app/services/authentication.py

```diff
@@ -5,16 +5,16 @@
 from app.utils.errors import AccessDenied
 
 
-def encode_token(email, salt="default-salt"):
+def encode_token(uuid, salt="default-salt"):
     serializer = URLSafeTimedSerializer(current_app.config["JWT_SECRET_KEY"])
-    return serializer.dumps(email, salt)
+    return serializer.dumps(uuid, salt)
 
 
 def decode_token(token, salt="default-salt", expiration=3600):
     serializer = URLSafeTimedSerializer(current_app.config["JWT_SECRET_KEY"])
     try:
-        email = serializer.loads(token, salt=salt, max_age=expiration)
-        return email
+        uuid = serializer.loads(token, salt=salt, max_age=expiration)
```

Comment:
don't need the local assign here.

`return serializer.loads(token, salt=salt, max_age=expiration)`